packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -831,7 +831,7 @@ void WebView::InitWebView() {
 void WebView::HandleMethodCall(
     const flutter::MethodCall<flutter::EncodableValue>& method_call,
     std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> result) {
-  if (!webview_instance_) {
+  if (!webview_instance_ || !method_call.arguments()) {
```

Comment:
Approving for now, but the overall logic should be improved and refactored soon. The `HandleMethodCall` method should not return without sending a result.